notebooks/naive_bayes.ipynb

```diff
@@ -774,7 +784,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.10.11"
+   "version": "3.12.3"
```

Comment:
This is probably related to your local setup. Please remove this from the PR ✌🏼 (similar for the other places below)

Reply:
Ah yes the problem was that I had executed some cells in those notebooks and so the stored metadata of the output changed. I have restored the files to their old output, so now the only change is the added code cell with the autoreload.